app/controllers/movies_controller.rb

```diff
@@ -1,24 +1,31 @@
 class MoviesController < ApplicationController
+  skip_before_action :doorkeeper_authorize!, only: %i[index show]
 
   def index
-    if self.validate?
-      render json: Movies::Representer.new(Movie.all).single.where(
-        cinema_hall_id: params[:cinema_hall_id]
-      ).order(title: :asc), except: [:created_at, :updated_at]
-    end
+      #link = CinemaHall.where(cinema_hall_id: params[:cinema_hall_id])
+      #throw(:abort) unless link 
+      if (params.has_key?(:cinema_hall_id))
+        movie = Movie.where(
+          cinema_hall_id: params[:cinema_hall_id]
+        ).order(title: :asc)
+      else
+        movie = Movie.all
```

Comment:
```suggestion
        movie = Movie.all.order(title: :asc)
```
to keep consistency?